src/main/java/core/basesyntax/repository/TransactionDB.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.repository;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionDB {
+    public static final List<FruitTransaction> transactions = new ArrayList<>();
+}
```

Comment:
You can make it like one Service "TransactionParseService" with method -
List<FruitTransaction> parse(String dataFromCsv);

in impl just parse data from file and save into List<FruitTransaction>
You did it in TransactionDaoImpl